src/spot.ts

```diff
@@ -1,6 +1,8 @@
 require("dotenv").load();
-import temp from "temp";
-import request from "request";
+import * as temp from "temp";
```

Comment:
For some reason synthetic imports aren't working, will have to circle back, maybe a config thing